.github/workflows/int-test-workflow.yml

```diff
@@ -0,0 +1,122 @@
+name: Integration Test WorkFlow
+on:
+  workflow_dispatch:
+    inputs:
+      name:
+        description: 'User Running the workflow'
+        # Default value if no value is explicitly provided
+        default: 'pdhanoya'
+        # Input has to be provided for the workflow to run
+        required: true    
+jobs:
+  smoke-tests:
```

Comment:
are we running smoke-tests here? or should it be int-tests? Smoke tests are not handled by smoke-test-workflow.yml?